community-operators/cert-utils-operator/cert-utils-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
-#! package-manifest: deploy/chart/catalog_resources/rh-operators/vaultoperator.v0.4.10.clusterserviceversion.yaml
```

Comment:
@raffaelespazzoli please delete this line if not needed and also fill checklist (copy and fill).

Reply:
I don't see that line in my PR....